### ISSUE TITLE
(pending) 302 redirect

### ISSUE THREAD

**Issue by howaboutsynergy** [ungoogled-software/ungoogled-chromium-archlinux#37]
Hey could any of you using ungoogled-chromium on archlinux, try about 10 shift+Refresh(button) on this page https://crates.io/crates/okofdb to see if you can reproduce [this issue](https://github.com/rust-lang/crates.io/issues/1865#issuecomment-542909049) whereby you sometimes see different content? it only happens with ungoogled-chromium and I'm trying to figure out if it's because of the extra patches that I've added on my version, or if it's some of the ungoogled-chromium patches instead. So if you can't reproduce it in 10 refreshes, it's probably my patches.

Basically, if you (ever) see only this, then it's ungoogled-chromium's patches:
```
About This Package
One Key One File Database
```
like this:
![Untitled](https://user-images.githubusercontent.com/49106389/66963157-47526c00-f073-11e9-8a93-57f8f1b48fb3.png)

**Comment by jstkdng** [comment]
Not under arch right now but can confirm the issue in ungoogled chromium under windows

Wed Oct 16 17:16:16 GMT-05:00 2019 howaboutsynergy <notifications@github.com>:

**Comment by wchen342** [comment]
I cannot reproduce this on Arch, but I'm on `77.0.3865.90-1` so I don't know whether a new version will trigger this.
Can it be a network problem?

**Comment by howaboutsynergy** [comment]
> Can it be a network problem?

definitely not a network problem

It works on Firefox btw, haven't tried upstream Chromium yet though.

**Comment by aksel** [comment]
I am not able to reproduce it on `77.0.3865.120-2`.
